.changeset/curly-chairs-speak.md

```diff
@@ -1,5 +0,0 @@
----
-'@shopify/polaris': minor
----
-
-Add a search field to filter ActionList that have more than 10 items
```

Comment:
@sam-b-rose I had you as the author of this file but I think we already have an entry for it in main.